src/nvim/ex_docmd.c

```diff
@@ -8180,3 +8180,18 @@ uint32_t get_cmd_argt(cmdidx_T cmdidx)
 {
   return cmdnames[(int)cmdidx].cmd_argt;
 }
+
+/// Check if a command is a mapping/abbreviation command by examining its handler function
+bool is_mapping_command(cmdidx_T cmdidx)
```

Comment:
afaict "mapping" is not used in any C function names. either "map" or "keymap".

```suggestion
bool is_map_cmd(cmdidx_T cmdidx)
```